depends/packages/native_pyqt5.mk

```diff
@@ -0,0 +1,18 @@
+package=native_pyqt5
+$(package)_version=5.4.1
+$(package)_download_path=http://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-$($(package)_version)
+$(package)_file_name=PyQt-gpl-$($(package)_version).tar.gz
+$(package)_sha256_hash=c6c33f392c0a2b0c9ec16531dc88fa9aee853e9fff10f0ad9d0e4777629fe79e
+$(package)_dependencies=native_python3 native_qt native_sip
+
+define $(package)_config_cmds
+  $($(package)_prefix)/bin/python3 configure.py --confirm-license --bindir=$($(package)_prefix)/bin --qmake=$($(package)_prefix)/bin/qmake-native --sip=$($(package)_prefix)/bin/sip --sip-incdir=$($(package)_prefix)/include/python3.4m --spec=linux-g++ --verbose
```

Comment:
Probably not entirely. I think the reason it works is that I am on Linux and this is a native package. Also Gitian uses Linux, so there are no problems with this in that case. But if you are using the depends system for a normal build and you are on OS X it probably won't work. So I'll fix that.